pipelines/controller/src/main/java/com/google/fhir/analytics/DataProperties.java

```diff
@@ -138,6 +143,7 @@ void validateProperties() {
       logger.info("Using FHIR-search mode since dbConfig is not set.");
     }
     Preconditions.checkState(!createHiveResourceTables || !thriftserverHiveConfig.isEmpty());
+    Preconditions.checkState(!createHiveResourceTables || createParquetDwh);
```

Comment:
I think there are more config sanity check that needs to be done, e.g., when we are not generating Parquet files, generation of views should be disabled as well.